client/dashboard/sites/staging-site-sync-modal/index.tsx

```diff
@@ -2,13 +2,57 @@ import {
 	Button,
 	ExternalLink,
 	Modal,
+	Icon,
 	__experimentalText as Text,
 	__experimentalHStack as HStack,
 	__experimentalVStack as VStack,
+	// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+	__experimentalInputControl as InputControl,
+	// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+	__experimentalInputControlPrefixWrapper as InputControlPrefixWrapper,
 } from '@wordpress/components';
 import { createInterpolateElement } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
+import { __, isRTL } from '@wordpress/i18n';
+import { chevronRight, chevronLeft } from '@wordpress/icons';
 import InlineSupportLink from '../../components/inline-support-link';
+import SiteBadge, { EnvironmentType } from '../site-badge';
+
+const DirectionArrow = () => {
+	return (
+		<div style={ { marginTop: '38px' } }>
+			<Icon
+				icon={ isRTL() ? chevronLeft : chevronRight }
+				style={ {
+					fill: '#949494',
+				} }
+			/>
+		</div>
+	);
+};
+
+interface EnvironmentLabelProps {
+	label: string;
+	environmentType: EnvironmentType;
+}
+
+const EnvironmentLabel = ( { label, environmentType }: EnvironmentLabelProps ) => {
+	return (
+		<VStack spacing={ 3 } style={ { flex: 1 } }>
+			<Text weight={ 500 }>{ label }</Text>
```

Comment:
`WIP: Header` in Figma should be translated to `<SectionHeader />` (`client/dashboard/components/section-header`). The font-size does not really match though -- it will be 15px instead of 13px. I think it's a known issue -- cc: @matt-west 

```suggestion
			<SectionHeader level={3} title={ label } />
```